feat: Extended the sdk-scheme-adapter documentation to be more complete and include the bulk enhancements design and test implementation (#371)
mojaloop/documentation: docs/technical/sdk-scheme-adapter/BulkEnhancements/SDKBulk-EventSourcing-Design.md

# SDK Support for Bulk Transfers - DDD and Event Sourcing Design
## Design overview

> This diagram summarizes the design role the SDK is performing.

Reviewer: ```suggestion
This diagram gives an overview of the SDK design.
```
